bbc/simorgh
# Remove remaining most watched and recent assets links from navs, update Turkish recommender title

Resolves JIRA n/a

Overall changes
======
Update navigation bar on Asian sites to replace links to /media/video pages with links to Video Topics
Update navigation bar on the Sinhala and Indonesia site to replace links to recent asset index with links to relevant topics to avoid the redirect
Update the Recommender title on the Turkish site

Code changes
======

- replace /media/video link with link to Video topic in the navigation section of:
  - src/app/lib/config/services/burmese.ts
  - src/app/lib/config/services/gujarati.ts
  - src/app/lib/config/services/hindi.ts
  - src/app/lib/config/services/marathi.ts
  - src/app/lib/config/services/punjabi.ts
  - src/app/lib/config/services/telugu.ts
  - src/app/lib/config/services/urdu.ts
  - src/app/lib/config/services/vietnamese.ts
  - src/app/lib/config/services/zhongwen.ts

- remove /media/video link form the navigation section of src/app/lib/config/services/ukchina.ts

- replace the links to recent asset index URLs wiht link to the relevabt topics in the navigation section of:
  - src/app/lib/config/services/indonesia.ts
  - src/app/lib/config/services/sinhala.ts

- update recommendationTitle string on src/app/lib/config/services/turkce.ts

- updated snapshots

Testing
======
1. Open the front pages of the following sites, **there should be no links to /{service}/media/video in the navigation**:
   - Burmese: last item should point to /burmese/topics/cj7v92le02qt
   - Gujarati: second item should point to /gujarati/topics/cpev7m0rry0t
   - Hindi: second to last item should point to /hindi/topics/cw9kv0kpxydt
   - Marathi: second to last item should point to /marathi/topics/cl29j0epz13t
   - Punjabi: second to last item should point to /punjabi/topics/cx12qmz6jm4t
   - Telugu: second to last item should point to /telugu/topics/cl29j0e3e2dt
   - Urdu: to last item should point to /urdu/topics/c1e0mzr3r2yt
   - Vietnamese: last item should point to /vietnamese/topics/cl29j0ekkvdt
   - Zhongwen Simp and Trad: second to last item should point to /zhongwen/simp/topics/ck5rznle6edt and /zhongwen/trad/topics/cgvl47l38e1t respectively

2. The UK China Trad and Sim navigation bars should contain one single item (主頁)
3. Open the front pages of these two sites, 
  - Indonesia: Dunia links should point to /indonesia/topics/cyz8evpl224t **not /indonesia/dunia**
  - Sinhala: second item should point to /sinhala/topics/cg7267dz901t  **not /sinhala/sri_lanka** third item should point to /sinhala/topics/c83plvepnq1t **not  /sinhala/world**
4. On bbc.com/turkce articles the Recomender title should be 'Önerilen haberler'





Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
